.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,2 @@
+paths-ignore:
+  - '**/vendor/*'
```

Comment:
IMHO: In the security point of view, it might be useful to scan the vendor packages as well. If the scan for vendor packages is found to be annoying, let's ignore them.

Reply:
How about testing vendor packages in another workflow?

Reply:
Ah, that might be a perfect solution!

Reply:
I also agree with @kmiya san 's opinion.
I'd like to see what it means to test in another workflow, does that mean it should be tested in the vendor's repository?

Reply:
No, I meant to add `codeql-vendor-config.yml`!